app/src/main/kotlin/com/ustadmobile/zim2xapi/AddxAPIStatementUseCase.kt

```diff
@@ -0,0 +1,45 @@
+package com.ustadmobile.zim2xapi
+
+import org.jsoup.Jsoup
+import java.io.File
+
+class AddxAPIStatementUseCase {
```

Comment:
Please add short class doc explaining what this does - e.g. adding scripts to generate xAPI statements for Perseus exercises.

Reply:
done